compact/tracking/definitions_craterlake.xml

```diff
@@ -41,19 +41,20 @@
 
     <comment> Main parameters for Barrel TOF layers </comment>
     <comment> Barrel TOF region </comment>
-    <constant name="BarrelTOF_zmin"          value="112.5*cm"/>
-    <constant name="BarrelTOF_zmax"          value="174.0*cm"/>
-    <constant name="BarrelTOF_radius_design" value="64.6*cm"/>
+    <constant name="BarrelTOF_zmin"          value="117.5*cm"/>
+    <constant name="BarrelTOF_zmax"          value="171.5*cm"/>
+    <constant name="BarrelTOF_radius_design" value="61.6*cm"/>
 
     <comment> Forward TOF region  </comment>
-    <constant name="ForwardTOFRegion_zmin"        value="174.0*cm" />
+    <constant name="ForwardTOFRegion_zmin"        value="ForwardPIDRegion_zmin" />
+    <constant name="ForwardTOFRegion_zmax"        value="ForwardTOFRegion_zmin + ForwardTOFRegion_length" />
     <constant name="ForwardTOFRegion_tan"         value="CentralTrackingRegionP_tan" />
     <constant name="ForwardTOFRegion_minR"        value="8*cm" />
-    <constant name="ForwardTOFRegion_maxR"        value="50*cm" />
+    <constant name="ForwardTOFRegion_maxR"        value="60*cm" />
     <constant name="ForwardTOF_xOffset"           value="-2.75*cm" />
     <constant name="ForwardTOF_det_height"        value="2.0*cm" />
     <constant name="ForwardTOF_length"            value="min(ForwardTOFRegion_length, 2.*cm)"/>
-    <constant name="ForwardTOF_zmin"              value="ForwardTOFRegion_zmin + ForwardTOFRegion_length - ForwardTOF_length - 1*cm"/>
+    <constant name="ForwardTOF_zmin"              value="ForwardTOFRegion_zmax - ForwardTOF_length - 1*cm"/>
```

Comment:
Not introduced here, but why is there a `- 1*cm` here?

Reply:
Nicolas would know the answer.

Reply:
@nschmidtALICE can you comment?